### ISSUE TITLE
Allow only single emoji-status assignment per pull request

### ISSUE THREAD

**Issue by bqqbarbhg** [emoji-status-consortium/emoji-status-spec#6]
I suggest allowing only a single status emoji assignment per pull request. This would facilitate focused discussion of the specific emoji assignment and help maintain the quality of the emoji representations.

This could be supported using GitHub's new [Issue and Pull Request templates](https://github.com/blog/2111-issue-and-pull-request-templates) to have preset fields to fill for the status code and emoji. On the other hand the template limits non-emoji-assignment pull requests.

**Comment by oamaok** [comment]
Since the templates aren't actually enforced but just plain Markdown, we could fit multiple templates in a single file or just let the non-emoji specific PRs/issues be of informal format.

**Comment by melonmanchan** [comment]
Agreed, I will update the CONTRIBUTING.md to reflect these new changes. Do either @bqqbarbhg or @oamaok have time for writing up a PR template?
